src/components/uploader/js/FileQueueView.js

```diff
@@ -223,8 +223,14 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     fluid.uploader.fileQueueView.refreshAfterUpload = function (that) {
-        var rowButtons = that.locate("fileIconBtn", that.locate("fileRows"));
-        rowButtons.prop("disabled", false);
+        var fileRows = that.locate("fileRows");
+        var rowButtons = that.locate("fileIconBtn", fileRows);
+        // only re-enable rowButtons for files that have not been uploaded.
+        rowButtons.each(function (index, rowButton) {
+            rowButton = $(rowButton);
```

Comment:
There's only one use of `$(rowButton)` in this scope. This line can be removed by moving `$(rowButton)` into line 232.